tests/views/test_addanythingdialog_arrange.py

```
@@ -55,3 +56,21 @@ def test_add_via_birth_with_two_parents(scene, view):
     assert son.scenePos() == QPointF(0, SPACING * 1.5)
     assert father.scenePos() == QPointF(-SPACING, 0)
     assert mother.scenePos() == QPointF(SPACING, 0)
+
+
+def test_Variable_add_to_existing_people(scene, view):
```

Ruff (0.13.2)

Redefinition of unused `view` from line 9

(F811)